src/components/molecules/Modal/Modal.tsx

```diff
@@ -48,8 +48,9 @@ const Modal: ModalComponent = ({ children, onRequestClose, showCloseButton = tru
     {showCloseButton && (
       <CrossIcon
```

Comment:
The current close button isn't accessible because it is not an interactive element. Let's wrap close icon into the button tag and also add it aria-label "Close" or hidden text "Close". This will make sure the element can be selected via keyboard and properly announce it via screen reader